### PR TITLE
MAT-7358 update twin.macro

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-7358](https://jira.cms.gov/browse/MAT-7358)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "single-spa": "^5.9.3",
     "single-spa-react": "^4.3.1",
     "styled-components": "^5.3.5",
-    "twin.macro": "^2.8.2"
+    "twin.macro": "^3.4.1"
```
Comment (adongare) [inline package.json]: Not 100% sure but I think `twin.macro` is not needed here

Comment (sb-cecilialiu) [reply package.json]: @adongare I would think so too, but it is used in GlobalStyles.tsx

Comment (sb-cecilialiu) [reply package.json]: @adongare The other spa has the same GlobalStyles.tsx, so I removed it from madie-util. Thanks!

**Review by adongare — APPROVED** [review]
